
(* This file was auto-generated based on "DynamicsParser.messages". *)

(* Please note that the function [message] can raise [Not_found]. *)

let message =
  fun s ->
    match s with
    | 0 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 57\n"
    | 1 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 56\n"
    | 2 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 55\n"
    | 18 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 54\n"
    | 19 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 53\n"
    | 20 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 52\n"
    | 22 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 51\n"
    | 23 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 50\n"
    | 115 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 49\n"
    | 26 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 48\n"
    | 108 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 47\n"
    | 109 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 46\n"
    | 27 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 45\n"
    | 28 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 44\n"
    | 29 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 43\n"
    | 61 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 42\n"
    | 62 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 41\n"
    | 93 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 40\n"
    | 94 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 39\n"
    | 96 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 38\n"
    | 97 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 37\n"
    | 63 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 36\n"
    | 87 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 35\n"
    | 88 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 34\n"
    | 66 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 33\n"
    | 67 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 32\n"
    | 81 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 31\n"
    | 82 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 30\n"
    | 68 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 29\n"
    | 69 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 28\n"
    | 70 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 27\n"
    | 65 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 26\n"
    | 30 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 25\n"
    | 31 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 24\n"
    | 32 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 23\n"
    | 33 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 22\n"
    | 34 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 21\n"
    | 35 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 17\n"
    | 48 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 14\n"
    | 49 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 13\n"
    | 51 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 12\n"
    | 54 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 11\n"
    | 55 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 10\n"
    | 52 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 9\n"
    | 50 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 8\n"
    | 3 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 7>\n"
    | 4 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 6>\n"
    | 9 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 5>\n"
    | 12 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 4>\n"
    | 13 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 3>\n"
    | 5 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 2>\n"
    | 8 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE 1>\n"
    | _ ->
        raise Not_found
